ingestion/src/metadata/ingestion/models/ometa_table_db.py

```diff
@@ -12,14 +12,14 @@
 
 from pydantic import BaseModel
 
+from metadata.generated.schema.api.data.createTable import CreateTableRequest
 from metadata.generated.schema.entity.data.database import Database
 from metadata.generated.schema.entity.data.databaseSchema import DatabaseSchema
 from metadata.generated.schema.entity.data.location import Location
-from metadata.generated.schema.entity.data.table import Table
 
 
 class OMetaDatabaseAndTable(BaseModel):
     database_schema: DatabaseSchema
     database: Database
-    table: Table
+    table: CreateTableRequest
```

Comment:
We have a ticket open to fix this at another point. Let's leave it for now

Reply:
Once we do that, we'll change the table, db and schema to the `Create` model. But let's not have them mixed